index.html

```diff
@@ -27,14 +33,132 @@
             <h1 class="main-header">I'm Hien <br> Glad to see you!</h1>
             <p class="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
             <div class="social">
-                <p>let's connect</p>
+                <h3>let's connect</h3>
                 <ul class="social-icon">
-                    <li><img src="./image/twister.png" alt="twister" class="contact-icon"></li>
-                    <li><img src="./image/phone.png" alt="phone" class="contact-icon"></li>
-                    <li><img src="./image/mail.png" alt="mail" class="contact-icon"></li>
-                    <li><img src="./image/facebook.png" alt="facebook" class="contact-icon"></li>
-                </ul>
+                    <li><a href="https://twitter.com/HIENPHAN0111"><i class="fa-brands fa-twitter contact-icon fa-xl"></i></a></li>
+                    <li><a href="https://www.linkedin.com/in/hien-phan-61097b256/"><i class="fa-brands fa-linkedin-in contact-icon fa-xl"></i></a></li>
+                    <li><a href="https://github.com/hienphan0111"><i class="fa-brands fa-github contact-icon fa-xl"></i></a></li>
+                    <li><a href="http://"><i class="fa-brands fa-angellist contact-icon fa-xl"></i></a></li>   
+                </ul>                
+            </div>
+        </section>
+        <section class="work">
+            <div class="project pr-1">
+                <div class="snap-shoot">
+                    <img src="./image/Snapshoot Portfolio-1.svg" alt="project 1">
+                </div>
+                <div class="info">
+                    <h2 class="pj-title pt-1">Tonic</h2>
+                    <ul class="meta">
+                        <li class="type">Canopy</li>
+                        <li class="catagories gray"><span class="dot"></span>Backend Dev</li>
+                        <li class="year gray"><span class="dot"></span>2015</li>
+                    </ul>
+                    <p class="project-des">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="langue">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button type="button" class="btn read-more"><a href="http://"> See Project</a></button>
```

Comment:
- [ ] [REQUIRED] Good job so far 👍 Please fulfill the request made by the [previous reviewer](), please implmen [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=hdw5gHzcXazz2Jky-0) `button` effects🙏🙏🙏

### Your design ❌ 
![image](https://user-images.githubusercontent.com/63915024/205326577-cbe13b6c-8500-4895-8053-00fe831af4d1.png)

### Figma design ✔️

![image](https://user-images.githubusercontent.com/63915024/205326618-50ce2212-0412-4099-b9a6-240cfaebe78b.png)